src/content/docs/zh-cn/guides/upgrade-to/v3.mdx

```diff
@@ -246,7 +248,7 @@ Astro v3.0 现在直接使用 `clsx` 来处理 `class:list`，它不支持去重
 
 ### 移除：将 `class:list` 作为 prop 传递
 
-在 Astro v2.x 中，[`class:list` 的值](/zh-cn/reference/directives-reference/#classlist) 通过 [`Astro.props['class:list']`](/zh-cn/reference/api-reference/#astroprops) 传递给组件。
+在 Astro v2.x 中，[`class:list` 的值](/zh-cn/reference/directives-reference/#classlist) 通过 [`Astro.props['class:list']`](/zh-cn/reference/api-reference/#props) 传递给组件。
```

Comment:
Keep this link as outdated `#astroprops`, because `api-reference` have not update.

When `api-reference` updated, I will back and fix it